cluster/addons/istio/noauth/istio.yaml

```diff
@@ -1,368 +1,1255 @@
-metadata:
-  name: istio-sidecar-injector-admin-role-binding-istio-system
-  labels:
-    k8s-app: istio
-    addonmanager.kubernetes.io/mode: Reconcile
-subjects:
-- kind: ServiceAccount
-  name: istio-sidecar-injector-service-account
-  namespace: istio-system
-roleRef:
-  kind: ClusterRole
-  name: istio-sidecar-injector-istio-system
-  apiGroup: rbac.authorization.k8s.io
----
-# Grant permissions to the CA.
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: istio-ca-role-binding-istio-system
-  labels:
-    k8s-app: istio
-    addonmanager.kubernetes.io/mode: Reconcile
-subjects:
-- kind: ServiceAccount
-  name: istio-ca-service-account
-  namespace: istio-system
-roleRef:
-  kind: ClusterRole
-  name: istio-ca-istio-system
-  apiGroup: rbac.authorization.k8s.io
----
-# Grant permissions to the Ingress controller.
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: v1
+kind: Namespace
 metadata:
-  name: istio-ingress-admin-role-binding-istio-system
+  name: istio-system
   labels:
     k8s-app: istio
     addonmanager.kubernetes.io/mode: Reconcile
-subjects:
-- kind: ServiceAccount
-  name: istio-ingress-service-account
-  namespace: istio-system
-roleRef:
-  kind: ClusterRole
-  name: istio-pilot-istio-system
-  apiGroup: rbac.authorization.k8s.io
+    istio-injection: disabled
 ---
-# Grant permissions to the sidecar.
-# TEMPORARY: the istioctl should generate a separate service account for the proxy, and permission
-# granted only to that account !
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
+# Source: istio/charts/mixer/templates/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
 metadata:
-  name: istio-sidecar-role-binding-istio-system
-  labels:
-    k8s-app: istio
-    addonmanager.kubernetes.io/mode: Reconcile
-subjects:
-- kind: ServiceAccount
-  name: default
+  name: istio-statsd-prom-bridge
   namespace: istio-system
-roleRef:
-  kind: ClusterRole
-  name: istio-sidecar-istio-system
-  apiGroup: rbac.authorization.k8s.io
----
-# Grant permissions to Mixer.
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: istio-mixer-admin-role-binding-istio-system
   labels:
     k8s-app: istio
     addonmanager.kubernetes.io/mode: Reconcile
-subjects:
-- kind: ServiceAccount
-  name: istio-mixer-service-account
-  namespace: istio-system
-roleRef:
-  kind: ClusterRole
-  name: istio-mixer-istio-system
-  apiGroup: rbac.authorization.k8s.io
+    app: istio-statsd-prom-bridge
+    chart: mixer-0.8.0
+    release: istio
+    heritage: Tiller
+    istio: mixer
+data:
+  mapping.conf: |-
 ---
-# Mixer
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: istio-mixer
+  name: istio-mixer-custom-resources
   namespace: istio-system
   labels:
     k8s-app: istio
     addonmanager.kubernetes.io/mode: Reconcile
+    app: istio-mixer
+    chart: mixer-0.8.0
+    release: istio
+    heritage: Tiller
```

Comment:
It's just a standard label recommended when using tiller to upgrade istio. If tiller is used by the user it will be clear from documentation that update of anything in istio-system will be reconciled regardless how it's updated.